index.js

```diff
@@ -3,7 +3,7 @@ module.exports = function (str, sep) {
 	if (typeof str !== 'string') {
 		throw new TypeError('Expected a string');
 	}
-	sep = sep || '_';
+	sep = typeof sep === 'undefined' ? '_' : sep;
```

Comment:
I made it that way to allow non-string stuff as separators. Anything passed is just converted to string in the next line.

It worked like that before, are you 100% sure? :)